### PR TITLE
Add documentation for iOS calendar component

### PR DESCRIPTION
**Pull request by dwroth**

### CODE REVIEW

**Comment by dwroth** [comment]
I cannot for the life of me figure out why the links at the top of this page are referencing the Dropdown component 🤔

File: www/src/pages/components/calendar/ios/index.mdx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+### Configuring the Calendar Height Modes
+
+![CalendarPicker - Height modes](/img/ios/calendar_height_modes.png 'CalendarPicker - Height Modes')
+
+CalendarPickers internally manage their own height – *they should never have a height constraint imposed upon them*.
+
+The CalendarPicker supports two height modes: `Fixed` and `Dynamic`. A fixed height CalendarPicker will always show the maximum number of rows that can displayed for a given calendar, even if the current month requires fewer rows. For a gregorian calendar, the maximum number of rows a month can need is six.
+
+If the `calendarHeightMode` is set to `dyanmic`, then the calendar will expand and collapse as as necessary to display the minimum number of rows possible for each month. After transitioning to a new month, a `dynamic` height CalendarPicker will update its own internal height constraint and notify its delegates of the update.
```
Comment (kevinmbeaulieu) [inline www/src/pages/components/calendar/ios/index.mdx:76]: nit: typo `dyanmic` => `dynamic`

**Review by kevinmbeaulieu — APPROVED** [review]
Looks great, just one typo to fix

**Review by kevinmbeaulieu — APPROVED** [review]

**Comment by danoc** [comment]
Thanks @dwroth and @kevinmbeaulieu! 🌮  🌮

**Pull request merged**